packages/crayons-extended/platform-ui/src/utils/utils.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { format } from './utils';
+
+describe('format', () => {
```

Comment:
please remove the unused test cases and add the test cases from crayons-core for the respective methods